src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -209,16 +209,17 @@ private enum Flags : byte
                                               this.IsStatic,
                                               hasInitializer: (_propertyFlags & Flags.HasInitializer) != 0,
                                               isCreatedForFieldKeyword: isCreatedForFieldKeyword,
-                                              isEarlyConstructed: isEarlyConstructed);
+                                              isEarlyConstructed: isEarlyConstructed,
+                                              diagnostics);
```

Comment:
>diagnostics [](http://example.com/codeflow?start=46&length=11)

It looks like there is a possible race here. The symbol that we create might "lose" on the next line, but it already recorded the diagnostics which is going to look like a duplicate. A "losing" symbol should have zero effect.